src/coreclr/inc/stresslog.h

```diff
@@ -225,7 +225,7 @@ class StressLog {
     CRITSEC_COOKIE lock;                    // lock
     uint64_t tickFrequency;         // number of ticks per second
     uint64_t startTimeStamp;        // start time from when tick counter started
-    FILETIME startTime;                     // time the application started
+    int64_t  startTime;                     // time the application started
```

Comment:
Should it be `uint64_t`?

Also, the comment should say that this is the timestamp in the Windows FILETIME precision (100ns since 01 January 1601).